### PR TITLE
Appointment request patch status field to 'fulfilled'

### PR DESCRIPTION
**Pull request by SudhirPai**
Description
Updated the Appointment 'patch' request to have 'fulfilled' status
With this enhancement, the Appointment resource can support updating the status of an appointment from booked to fulfilled.
--------------------------------------------**Screenshot Before Merge:**---------------------------------------

image
image

--------------------------------------------**Expected Screenshot After Merge:**-------------------------------
(changes: a new status 'fulfilled' is added)
image
image

----
PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

**Comment by sydneanschutz** [comment]
You also changed a line in the overview. can u provide screenshots of that as well please

File: content/millennium/r4/scheduling/appointment.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ When updating an appointment, the resource provides the ability to change the [A
 * From Proposed to Booked, or Cancelled
 * From Booked to Arrived, Checked-In, or Cancelled
 * From Arrived to Checked-In, or Cancelled
-* From Checked-In to Cancelled
+* From Checked-In to Cancelled or Fulfilled
```
Comment (MadhurRaj) [inline content/millennium/r4/scheduling/appointment.md:27]: let's make this
```
From Checked-In to Fulfilled, or Cancelled
```

to match the others.

**Review by MadhurRaj — CHANGES_REQUESTED** [review]

**Review by exopsRamba — APPROVED** [review]

File: content/millennium/r4/scheduling/appointment.md (hunk 1 above)
Comment (SudhirPai) [reply content/millennium/r4/scheduling/appointment.md]: https://github.com/cerner/fhir.cerner.com/pull/464/commits/3207a3c485e4126efcfe5a5dfe15cbc69b3cad34

**Review by sydneanschutz — APPROVED** [review]

**Review by MadhurRaj — APPROVED** [review]

**Comment by exopsRamba** [comment]
Screen Shot 2020-09-18 at 10 05 30 AM
Validated on live.

**Pull request merged**